spec/index.bs

```diff
@@ -8352,6 +8391,18 @@ enum GPUBlendFactor {
         <tr>
             <td><dfn>"one-minus-constant"</dfn>
             <td><code>(1 - R<sub>const</sub>, 1 - G<sub>const</sub>, 1 - B<sub>const</sub>, 1 - A<sub>const</sub>)</code>
+        <tr>
+            <td><dfn>"src1"</dfn>
+            <td><code>(R<sub>src1</sub>, G<sub>src1</sub>, B<sub>src1</sub>, A<sub>src1</sub>)</code>
+        <tr>
+            <td><dfn>"one-minus-src1"</dfn>
+            <td><code>(1 - R<sub>src1</sub>, 1 - G<sub>src1</sub>, 1 - B<sub>src1</sub>, 1 - A<sub>src1</sub>)</code>
+        <tr>
+            <td><dfn>"src1-alpha"</dfn>
+            <td><code>(A<sub>src1</sub>, A<sub>src1</sub>, A<sub>src1</sub>, A<sub>src1</sub>)</code>
+        <tr>
+            <td><dfn>"one-minus-src1-alpha"</dfn>
+            <td><code>(1 - A<sub>src1</sub>, 1 - A<sub>src1</sub>, 1 - A<sub>src1</sub>, 1 - A<sub>src1</sub>)</code>
```

Comment:
It would be good if this table added a column to indicate which feature is required to use the blend factor, like the [Packed Formats](https://gpuweb.github.io/gpuweb/#packed-formats) table. (Column can be blank if the blend factor does not requires a feature.

Reply:
Repository